docs/app/file-library.md

```diff
@@ -1,137 +1,209 @@
-- **Filename (Disk)** — LOCKED, this is the actual name of the file in storage
-- **Filename (Download)** — This allows you to set the name of the file when it is downloaded
+When a file is clicked in the File Listing Page, the File Details Page appears. This is a custom form for viewing assets
+and embeds, with core Fields included out-of-the-box (see below), and the ability for administrators to add additional
+custom Fields. This page has the same features and functionality as the
+[Item Details Page](/getting-started/glossary/#items).
 
-The file sidebar also includes the following details:
+![Files](https://cdn.directus.io/docs/v9/app-guide/file-library/file-library-20220305A/files-20220305A.webp)
 
-- **Type** — The MIME type of the file, displayed in the App as a formatted media type
-- **Dimensions** — (Images Only) The width and height of the image in pixels
-- **Size** — The file-size the asset takes up within the storage adapter
-- **Created** — The timestamp of when the file was uploaded to the project
-- **Owner** — The Directus user that uploaded the file to the project
-- **Modified** - The timestamp of when the file was last modified
-- **Edited By** — The Directus user that modified the file
-- **Folder** — The current parent folder that contains the file
-- **Metadata** — Metadata JSON dump of the file's EXIF, IPTC, and ICC information
+### Action Buttons
 
-#### Action Buttons
+Notice the following Buttons in the Header:
 
-- <span mi btn >check</span> — Saves any edits made to the file
-- <span mi btn sec>tune</span> — Refer [Editing an Image](#editing-an-image)
-- <span mi btn sec>save_alt</span> — Downloads the file to your current device
-- <img src="../assets/buttons/move-file.png" alt="Move to Folder" height="32" style="vertical-align: middle; margin: 2px 0; box-shadow: none">
-  — Move the file to another folder
-- <span mi btn dngr>delete</span> — Permanently removes the file and its metadata. This action is permanent and can not
+- <span mi btn >check</span> – Saves any edits made to the file.
+- <span mi btn sec>tune</span> – Please see the [Edit an Image](#edit-an-image) to learn more.
+- <span mi btn sec>save_alt</span> – Downloads the file to your current device.
+- <span mi btn sec>drive_file_move</span> – Moves selected file(s) to another folder.
+- <span mi btn dngr>delete</span> – Permanently removes the file and its metadata. This action is permanent and cannot
   be undone.
 
-## Editing an Image
+::: tip Can't Delete Files Linked to Items
 
-Clicking the <span mi btn sec>tune</span> action button on the File Detail opens an image editor for rotating, cropping,
-or mirroring the file.
+Directus will not allow you to delete a File until you remove it from any and all related Items.
 
-1. From the **File Library**, click a file to open it's detail page
-2. Click the <span mi btn sec>tune</span> button in the top right to open the image editor
+:::
```

Comment:
Lets update this, as it's not technically true. You can configure the relational constraint of your file field to SET NULL or CASCADE when the file is deleted. That way, the referencing field will get updated on the fly when the image is deleted.

Reply:
This is a duplicate of the above comment.